### PR TITLE
Add romanian translations

### PR DESCRIPTION
**Pull request by mihaianghel**
New folder /ro was added in version 1.4 and the href for this page was added to the existing translations index.htm files.

### CODE REVIEW

**Comment by CoralineAda** [comment]
This is awesome! I would like another native speaker to review the translation before merging. Thanks so much!

**Comment by mihaianghel** [comment]
Hi @CoralineAda , I could suggest user https://github.com/MirelaI for reviewing this. She's a native  Romanian speaker.

File: version/1/4/ro/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+* Respectarea punctelor de vedere si a experientelor alternative
+* Acceptarea criticilor constructive intr-un mod civilizat      
+* Orientarea catre binele comunitatii                           
+* Exprimarea empatiei fata de ceilalti membrii ai comunitatii   
+
+Exemple de comportament inadecvat ale unui participant includ:                   
+
+* Folosirea unor expressii, imagini sau aluzii cu tenta sexuala             
+* Adresarea unor comentarii cu caracter de batjocura sau insulta. Atacuri de natura
+politica sau personala.
```
Comment (MirelaI) [inline version/1/4/ro/code_of_conduct.md]: No need for new line here

File: version/1/4/ro/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+probleme sau orice alte contributii care nu respecta acest Cod de Conduita sau de a
+exclude temporar sau permanent oricare contribuitor pentru actiuni pe care le
+considera neadecvate, amenintatoare, ofensatoare sau periculoase.
+
+## Scop                                                                      
+                                                                             
+Acest Cod de Conduita se applica atat in spatiul privat cat si in cel public atunci
+cand o persoana reprezinta proiectul sau comunitatea sa. Exemple de reprezentare a
+proiectului sau a comunitatii includ folosirea unei adrese oficiale de email a
+proiectului, postarea prin intermediul unui cont dintr-un mediu de socializare sau
```
Comment (MirelaI) [inline version/1/4/ro/code_of_conduct.md]: Maybe better: "Postarea pe retele de socializare folosind un cont official"

File: version/1/4/ro/code_of_conduct.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+prezenta ca un reprezentant numit la un eveniment 'online' sau 'offline'. Reprezentarea
```
Comment (MirelaI) [inline version/1/4/ro/code_of_conduct.md]: "Participarea la un eveniment 'online' sau 'offline' ca reprezentant delegat"

**Comment by mihaianghel** [comment]
Thanks @MirelaI . @CoralineAda  I think this can be merged.

**Comment by CoralineAda** [comment]
Thank you so much!

**Pull request merged**